README.md

```diff
@@ -1,2 +1,59 @@
-# DevOps-Azure-plugin
-Leapwork Integration with DevOps Azure
+# Leapwork - Azure DevOps and TFS Integration  
+
+# Streamline your test automation  
+A significant part of running automated cases is to monitor, inspect and react on the results. Integration with test management and bug tracking systems enables to you streamline your test automation and development cycles so that not a single flaw of the system is left unnoticed.  
```

Comment:
we have updated Readme file. use the template from Jenkins or TeamCity repos

Reply:
Updated readme file as per Jenkins template